.github/workflows/notebook-local-verify.yaml

```
@@ -0,0 +1,38 @@
+name: Docker notebook tests
```

nit:

```suggestion
name: Test notebooks with LocalProvider
```

or soemthing like that

---

was already done before I hit "comment" :smile:

---

I've just noticed.